using-mobile-extensions/mobile-core/lifecycle/README.md

```diff
@@ -3,7 +3,7 @@
 {% hint style="warning" %}
 In version 4 of the iOS SDK, this implementation was completed automatically.
 
-When upgrading to the Experience Platform SDK, you must add code to continue collecting Lifecycle metrics.
+The Experience Platform SDK will not collect Lifecycle metrics for you automatically. If you wish to continue collecting Lifecycle metrics, you must add code to your app.
```

Comment:
Please reword so that this section reads as follows:
"The Experience Platform SDK will not automatically collect Lifecycle metrics for you . To continue collecting Lifecycle metrics, you must add code to your app."